ccic/data/training_data.py

```diff
@@ -255,7 +255,7 @@ def _load_sample(self, filename):
             data["tiwp"] = data["tiwp"] * 1e-3
             tiwp_fpavg = load_output_data(data, "tiwp_fpavg", 1e-6, 1e-3, self.rng)
             tiwp = load_output_data(data, "tiwp", 1e-6, 1e-3, self.rng)
-            tiwc = load_output_data(data, "tiwc", 1e-6, 1e-3, self.rng)
+            tiwc = load_output_data(data, "tiwc", 1e-10, 1e-7, self.rng)
```

Comment:
This change was because the distribution of IWC values extends to smaller sizes than 1e-3. That was used for the training of the most recent retrieval.